app/javascript/packs/application.js

```diff
@@ -79,5 +79,7 @@ require("trix")
 require("@rails/actiontext")
 // prevent adding attachments via drag and drop
 document.addEventListener("trix-file-accept", function(event) {
-  event.preventDefault();
+  if(event.target.parentNode.parentNode.classList.contains('no-attachments')) {
```

Comment:
Try setting the event listener only on the relevant elements, not on the document

Reply:
I think it's pretty ok the way it is.